src/Test/Integration/KV/FetchPreflistTests.cs

```diff
@@ -16,7 +16,7 @@
 // under the License.
 // </copyright>
 
-namespace Test.Integration.CRDT
```

Comment:
Yeah, just noticed that out of the blue.  I think there was a typo in one of the copyright headers too somewhere that I fixed.